### ISSUE TITLE
Only get options for indian songs and artists when adding metadata

### ISSUE THREAD

**Issue by DavidUzielGH** [deepjyoti30/ytmdl#77]
Using ytmdl to search for the songs:
- Lights by Interpol
- Are we ready by Two Door Cinema Club
- Ready to start by Arcade fire

The only options ytmdl provided for metadata where Indian artists / songs that resembled in some way either the song name or the artist name.

The arguments used for these searches where:
```
ytmdl --song Lights --artist Interpol Lights
ytmdl --song "Ready to start" --artist "Arcade Fire" Ready to start
ytmdl --song "Are we ready" --artist "Two door cinema club" Are we ready

```
Below some screenshots of the types of results I got.
![2020-06-17-163735_1920x1080_escrotum](https://user-images.githubusercontent.com/29080200/84953672-91379a00-b0b9-11ea-8b2d-d0f84db54fd0.png)
![2020-06-17-164005_1920x1080_escrotum](https://user-images.githubusercontent.com/29080200/84953674-91d03080-b0b9-11ea-9131-1cc2c2a3504a.png)
![2020-06-17-164834_1920x1080_escrotum](https://user-images.githubusercontent.com/29080200/84953962-1e7aee80-b0ba-11ea-8e90-03ec368bf68e.png)

**Comment by deepjyoti30** [comment]
Hey @DavidUzielGH thanks for reporting that. It's kind of an weird issue because YTMDL itself doesn't do any filtering based on the region so it might be the sources itself that are returning this data.

Anyways, I'll take a look into it.

**Comment by jepace** [comment]
I have the same issue.  As a test, I picked "Telephone" (by Lady Gaga with Beyonce), which surely must be the most popular "Telephone" song?  Downloaded the song fine, but is was not in the list of 33 possible matches.

**Comment by danimateo** [comment]
I fixed this and added some other stuff as well. Check #85.

**Comment by BoBomann18** [comment]
I also have this problem, I downloaded ytmdl today so I have the newest version.

**Comment by deepjyoti30** [comment]
@BoBomann18 A temporary fix is to install the 1.5.5 version of itunespy, do it by the following command

```console
pip install itunespy==1.5.5
```

This will fix the issue for the time being.
